src/pages/OPP_MatchingDonationsBTN.page

```diff
@@ -1,3 +1,3 @@
-<apex:page standardController="Opportunity" extensions="OPP_MatchingDonationsBTN_CTRL" action="{!MatchingDonationPage}">
-    <apex:pageMessages />
+<apex:page standardController="Opportunity">
+    This button has been deprecated.
```

Comment:
Is it worth using a label so it's translatable?

Reply:
I was wondering that myself. I would think not since the message is temporary and probably wouldn't be translated before it was ultimately deleted.

Reply:
Keep in mind that even once we delete it from the managed package, my understanding is that the VF page will remain in existing subscriber orgs until an admin deletes it.

Reply:
With buttons and fields we usually add DEPRECATED to the label before deletion, so I just made that change to the Visualforce page as well. However, let me know if you really feel this needs to be translatable and I can add a label. Maybe a generic DEPRECATED label that can be used down the road? I also created a PR in the underlying package to remove the button that calls this page.

Reply:
I like the idea of a generic "This component has been deprecated" label but I don't personally feel strongly about what we do here.